Documentation/dev/libvirt-howto.md

```diff
@@ -7,16 +7,57 @@ for operator development.
 ### 1. One-time setup
 It's expected that you will create and destroy clusters often in the course of development. These steps only need to be run once (or once per RHCOS update).
 
-#### 1.1 Pick a name and ip range
+Before you begin, install the [general installer development
+dependencies](dependencies.md).
```

Comment:
It would be nice to have a link like this from `CONTRIBUTING.md` too.

Reply:
Sure, I'll add it.